package.json

```diff
@@ -8,7 +8,8 @@
     "build": "tsc && vite build",
     "preview": "vite preview",
     "lint": "eslint src",
-    "format": "prettier --write . && yarn lint --fix",
+    "prettier": "prettier --check src",
+    "format": "prettier --write src && yarn lint --fix",
```

Comment:
Format should only affect white space and eslint fixers do more than that. They can break code. Please remove eslint from format.

Also, why only src? A bunch of non-src files were reformatted in this PR. Please omit those changes or make this continue to reformat them.

Reply:
Done